Cargo.toml

```diff
@@ -166,25 +166,25 @@ wadm = { version = "0.7", default-features = false }
 walkdir = { version = "2", default-features = false }
 warp = { version = "0.3", default-features = false }
 warp-embed = { version = "0.4", default-features = false }
-wascap = { version = "0.11", path = "./crates/wascap", default-features = false }
-wash-cli = { version = "0.22", path = "./crates/wash-cli", default-features = false }
-wash-lib = { version = "0.13", path = "./crates/wash-lib", default-features = false }
+wascap = { version = "0", path = "./crates/wascap", default-features = false }
+wash-cli = { version = "0", path = "./crates/wash-cli", default-features = false }
+wash-lib = { version = "0", path = "./crates/wash-lib", default-features = false }
```

Comment:
The reason why we explicitly versioned these from `*` was so that crates.io would publish dependents and know what version to use, does using `version = "0"` let this happen? If I remember correctly the dry run didn't catch it before

Reply:
It should, but there's really only one way to find out if dry run does not catch it